testing/environments/Makefile

```diff
@@ -10,6 +10,10 @@ stop:
 	${BASE_COMMAND} stop
 	${BASE_COMMAND} rm -f
 
+
+up:
```

Comment:
I recommend also to run the build command before running up, to make sure you always have the most recent environment.